src/config/cfg_rules.ini

```diff
@@ -474,13 +474,15 @@ option = ad_use_ldaps
 option = ad_allow_remote_domain_local_groups
 
 # IPA provider specific options
+option = ipa_access_order
 option = ipa_anchor_uuid
 option = ipa_automount_location
 option = ipa_backup_server
 option = ipa_deskprofile_refresh
 option = ipa_deskprofile_request_interval
 option = ipa_deskprofile_search_base
 option = ipa_subid_ranges_search_base
+option = ipa_access_order
```

Comment:
I guess there is no reason to add it twice?

Reply:
You guess right.